src/carma-platform/test/test_parse_ros2_bags.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from pathlib import Path
+from unittest.mock import patch, MagicMock
+from parse_ros2_bags import extract_mcap_data
+
+@pytest.fixture
+def mock_mcap_path():
+    return Path("/path/to/mock.mcap")
```

Comment:
Do we need `mock.mcap` stored in the git repo for these unit tests to run?

Reply:
Nope. We don't need it as the `with patch` call just mocks all functions and just returns values expected so we can return any values as if real mcap exists.

Reply:
Got it, thanks for the clarification!